README.md

```diff
@@ -376,6 +396,11 @@ conjur::ssl_certificate: |
   -----END CERTIFICATE-----
 ```
 
+**Note that establishment of identity using host factory tokens through this module is
+deprecated and will be removed in the next major version. If you are still using this
+feature, please use the [v2](https://github.com/cyberark/conjur-puppet/tree/v2) branch
++of this project or a release version `<=3.0.0`**
+
 ##### Pre-established host identity (Conjur Enterprise v4 only)
 
```

Comment:
Maybe this branch should also have a deprecation note for Conjur Enterprise here?